error.go

```diff
@@ -213,17 +213,9 @@ func (e *Error) messageFromProperties() string {
 	if e.printablePropertyCount == 0 {
 		return ""
 	}
-	uniq := make(map[Property]struct{}, e.printablePropertyCount)
 	strs := make([]string, 0, e.printablePropertyCount)
-	for m := e.properties; m != nil; m = m.next {
-		if !m.p.printable {
-			continue
-		}
-		if _, ok := uniq[m.p]; ok {
-			continue
-		}
-		uniq[m.p] = struct{}{}
-		strs = append(strs, fmt.Sprintf("%s: %v", m.p.label, m.value))
+	for k, v := range e.GetAllPrintableProperties() {
```

Comment:
The problem here is that `map`s in Go don't preserve insertion order. Therefore the strings in `strs` will appear in a kind of randomized order. I don't see any particular problem with that now though but there is no reason to break ordering here.

Reply:
I agree that it might be a problem, but on the other hand I copied that from the existing implementation. It was there, just took it.
I do not see a problem with making it ordered, but currently, it's iterating over `properties` map, which seems to be a linked list. If that is the case, except duplicated names (again, limited use case currently, as noted in other comments, which for me is still an open question to address), the order should be mostly the same on insertion. Now, the iteration of maps is randomized, but I believe all Go devs know that...

Reply:
Still, there is indeed a regression here.
Presently, properties are being printed in reverse order of how they were put into error (skipping overwritten earlier values).
After currently proposed change, they will be printed in random order. This needlessly adds confusion. 

This must be fixed. For example, a private method may be introduced to return a resulting slice of a key-value pairs, and then `messageFromProperties ` and the new method would use this result a but differently.